code/modules/reagents/chemistry/reagents.dm

```diff
@@ -22,7 +22,6 @@
 	var/volume = 0									//pretend this is moles
 	var/color = "#000000" // rgb: 0, 0, 0
 	var/can_synth = TRUE // can this reagent be synthesized? (for example: odysseus syringe gun)
-	var/can_synth_debug = TRUE // can this reagent be synthesized by the debug chem synthesizer?
```

Comment:
@81Denton why

Reply:
don't drink and code kids

Reply:
TBF I too would expect all chems to be synth'd by the debug chem synthesizer.